scripts/onboarding/aks/onboarding-msi-azure-policy/README.md

```diff
@@ -0,0 +1,10 @@
+You can create the policy definition using a command like :
+
+```az policy definition create --name "CITest" --display-name "CITest" --mode Indexed --metadata version=1.0.0 category=Kubernetes --rules .\azure-policy.rules.json --params .\azure-policy.parameters.json```
+
+**NOTE**
+
+- Please download all files under AddonPolicyTemplate folder before running the policy template.
+- After creating the policy definition through the above command, go to Azure portal -> Policy -> Definitions and select the definition you just created.
+- Click on 'Assign' and then go to the 'Parameters' tab and fill in the details. Then click 'Review + Create'.
```

Comment:
For Policy assignment, you can have the instructions similar to this - https://learn.microsoft.com/en-us/azure/azure-monitor/containers/container-insights-enable-aks-policy#assign-a-policy-definition-to-a-specified-scope-1